app/src/main/java/at/aau/serg/activities/MainActivity.kt

```diff
@@ -46,18 +46,16 @@ class MainActivity : AppCompatActivity() {
         ) {
             this.startActivity(Intent(this, LoginActivity::class.java))
         } else {
-            Authentication.getMe(
+            Authentication.tokenValid(
```

Comment:
I would prefer to keep the getMe instead of tokenValid the request returns not really if the token is valid. It returns data about the player. Token validation is only a side effect because the request is behind the auth middleware

Reply:
Yes we can rename it, but I removed the function since it was completely duplicated

Reply:
Renamed it to `getCurrentUser`